src/lib/ngx-jira-issue-collector.component.ts

```diff
@@ -66,6 +66,9 @@ export class NgxJiraIssueCollectorComponent implements OnInit {
   }
 
   emitMessage = (): void => {
+    if (!this.iframe) {
+      return;
+    }
```

Comment:
To Handle exception https://github.com/CatalystOne/ngx-jira-issue-collector/pull/66/files#diff-9095ccf56b9e876f1813e3ec6daf0727a80bfe7e04fe2da1f69a25635160876dR77

Reply:
Were we able to figure out why the iframe reference is undefined? Since it is not supposed to be that way in the first place.
Or were we able to figure out why this emitMessage function is triggered twice?

Reply:
I know you must have gone through this: https://stackoverflow.com/questions/10781880/dynamically-created-iframe-triggers-onload-event-twice#:~:text=The%20iframe%20onload%20event%20will,codes%20in%20the%20following%20way.

It seems **load**(culprit) event gets called twice, first when iframe is attaching to DOM, & other when it is actually loaded. We are not interested when iFrame is not actually loaded(first case). So, failing fast.

Above just tries to handle situation defensively.

Reply:
![image](https://user-images.githubusercontent.com/59651629/117317042-c6ea9a80-aea6-11eb-8ad3-86d133f76064.png)
![image](https://user-images.githubusercontent.com/59651629/117317267-fc8f8380-aea6-11eb-9052-bbc60d8ca1a4.png)

I am attaching 2 pics: 
1st case is when 1st invocation of load event happens. At this place, it has resolved **src** object but not attached to DOM yet.
2nd case is when second invocation happens, you see **src** is attached in DOM.

May be that is reason, **load** event is invoking 2 times